acquia_lift.admin.wizard.inc

```diff
@@ -577,34 +685,78 @@ function acquia_lift_personalize_campaign_wizard_targeting_submit(&$form, &$form
     return;
   }
   module_load_include('inc', 'acquia_lift', 'acquia_lift.admin');
+  $targeting = array();
+  $audience_count = 0;
+  // Determine the values for the "Everyone else" audience
+  $else = array(
```

Comment:
Didn't even think of that... but I'm certainly not married to it...was just lazy to type out $everyone_else.  But now that you said it I'll read it that way the rest of the time I'm working on it... so I'll change it.